### PR TITLE
Fix no-data error page rendering

### PR DESCRIPTION
**Pull request by ajacksified**
:eyeglasses: @curioussavage 

Specifically for: https://github.com/reddit/snoode/pull/56

This helps renders on error pages for API timeouts, which won't have data.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "semver": "^5.0.3",
     "simple-oauth2": "0.2.1",
     "snoode": "1.11.4",
-    "superagent": "1.3.0",
+    "superagent": "1.4.0",
```
Comment (curioussavage) [inline package.json]: so why don't we bump snoode to 1.4?

**Comment by curioussavage** [comment]
:nail_care:

File: package.json (hunk 1 above)
Comment (ajacksified) [inline package.json]: Whoops, reverting.

**Comment by curioussavage** [comment]
:fish:

**Pull request merged**